Can't find config file in

The docs state that 

> If -c is not provided, yamllint will look for a configuration file in the following locations (by order of preference): 
>    [...]
>    ~/.config/yamllint/config

but that doesn't seem to work:

```
$ yamllint -c ~/.config/yamllint/config example.yml 
$ yamllint  example.yml 
example.yml
  3:12      error    too many spaces after colon  (colons)
  4:12      error    too many spaces after colon  (colons)
  5:12      error    too many spaces after colon  (colons)
  6:12      error    too many spaces after colon  (colons)
  7:12      error    too many spaces after colon  (colons)
  8:12      error    too many spaces after colon  (colons)
  10:12     error    too many spaces after colon  (colons)
  11:12     error    too many spaces after colon  (colons)
  12:12     error    too many spaces after colon  (colons)
  13:8      error    trailing spaces  (trailing-spaces)
```

Using yamllint 1.2.1 as installed via official Ubuntu Xenial repo.

--- a/docs/configuration.rst
+++ b/docs/configuration.rst
@@ -14,11 +14,11 @@ To use a custom configuration file, use the ``-c`` option:
 If ``-c`` is not provided, yamllint will look for a configuration file in the
 following locations (by order of preference):
 
-- ``.yamllint``, ``.yamllint.yaml`` or ``.yamllint.yml`` in the current working
-  directory
-- the file referenced by ``$YAMLLINT_CONFIG_FILE``, if set
-- ``$XDG_CONFIG_HOME/yamllint/config``
-- ``~/.config/yamllint/config``
+- a file named ``.yamllint``, ``.yamllint.yaml``, or ``.yamllint.yml`` in the
+  current working directory
+- a filename referenced by ``$YAMLLINT_CONFIG_FILE``, if set
+- a file named ``$XDG_CONFIG_HOME/yamllint/config`` or
+  ``~/.config/yamllint/config``, if present
 
 Finally if no config file is found, the default configuration is applied.
 

